src/TRoom.cpp

```diff
@@ -760,89 +725,208 @@ void TRoom::auditExits( const QHash<int, int> roomRemapping )
-               exitWeightsCopy, exitStubsCopy, exitLocksCopy, doorsCopy,
-               customLinesCopy, customLinesColorCopy, customLinesStyleCopy, customLinesArrowCopy,
-               roomRemapping );
-
-    auditExit( south, DIR_SOUTH, tr( "South" ), QStringLiteral( "s" ), QStringLiteral( "S" ),
-               exitWeightsCopy, exitStubsCopy, exitLocksCopy, doorsCopy,
-               customLinesCopy, customLinesColorCopy, customLinesStyleCopy, customLinesArrowCopy,
-               roomRemapping );
-
-    auditExit( southeast, DIR_SOUTHEAST, tr( "Southeast" ), QStringLiteral( "se" ), QStringLiteral( "SE" ),
-               exitWeightsCopy, exitStubsCopy, exitLocksCopy, doorsCopy,
-               customLinesCopy, customLinesColorCopy, customLinesStyleCopy, customLinesArrowCopy,
-               roomRemapping );
-
-    auditExit( southwest, DIR_SOUTHWEST, tr( "Southwest" ), QStringLiteral( "sw" ), QStringLiteral( "SW" ),
-               exitWeightsCopy, exitStubsCopy, exitLocksCopy, doorsCopy,
-               customLinesCopy, customLinesColorCopy, customLinesStyleCopy, customLinesArrowCopy,
-               roomRemapping );
-
-    auditExit( east, DIR_EAST, tr( "East" ), QStringLiteral( "e" ), QStringLiteral( "E" ),
-               exitWeightsCopy, exitStubsCopy, exitLocksCopy, doorsCopy,
-               customLinesCopy, customLinesColorCopy, customLinesStyleCopy, customLinesArrowCopy,
-               roomRemapping );
-
-    auditExit( west, DIR_WEST, tr( "West" ), QStringLiteral( "w" ), QStringLiteral( "W" ),
-               exitWeightsCopy, exitStubsCopy, exitLocksCopy, doorsCopy,
-               customLinesCopy, customLinesColorCopy, customLinesStyleCopy, customLinesArrowCopy,
-               roomRemapping );
-
-    auditExit( up, DIR_UP, tr( "Up" ), QStringLiteral( "up" ), QStringLiteral( "UP" ),
-               exitWeightsCopy, exitStubsCopy, exitLocksCopy, doorsCopy,
-               customLinesCopy, customLinesColorCopy, customLinesStyleCopy, customLinesArrowCopy,
-               roomRemapping );
-
-    auditExit( down, DIR_DOWN, tr( "Down" ), QStringLiteral( "down" ), QStringLiteral( "DOWN" ),
-               exitWeightsCopy, exitStubsCopy, exitLocksCopy, doorsCopy,
-               customLinesCopy, customLinesColorCopy, customLinesStyleCopy, customLinesArrowCopy,
-               roomRemapping );
-
-    auditExit( in, DIR_IN, tr( "In" ), QStringLiteral( "in" ), QStringLiteral( "IN" ),
-               exitWeightsCopy, exitStubsCopy, exitLocksCopy, doorsCopy,
-               customLinesCopy, customLinesColorCopy, customLinesStyleCopy, customLinesArrowCopy,
-               roomRemapping );
-
-    auditExit( out, DIR_OUT, tr( "Out" ), QStringLiteral( "out" ), QStringLiteral( "OUT" ),
-               exitWeightsCopy, exitStubsCopy, exitLocksCopy, doorsCopy,
-               customLinesCopy, customLinesColorCopy, customLinesStyleCopy, customLinesArrowCopy,
-               roomRemapping );
+    auditExit(north,
```

Comment:
:-1: I think the one per line is not helpful here - OTOH I was the one who put them as they were so I may not be entirely impartial...!